private/buf/cmd/buf/command/beta/registry/webhook/webhookcreate/webhookcreate.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webhookcreate
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/bufbuild/buf/private/buf/bufcli"
+	registryv1alpha1 "github.com/bufbuild/buf/private/gen/proto/go/buf/alpha/registry/v1alpha1"
+	"github.com/bufbuild/buf/private/pkg/app/appcmd"
+	"github.com/bufbuild/buf/private/pkg/app/appflag"
+	"github.com/bufbuild/buf/private/pkg/command"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+const (
+	ownerFlagName        = "owner"
+	repositoryFlagName   = "repository"
+	callbackURLFlagName  = "callback_url"
+	webhookEventFlagName = "event"
+)
+
+// NewCommand returns a new Command
+func NewCommand(
+	name string,
+	builder appflag.Builder,
+) *appcmd.Command {
+	flags := newFlags()
+	return &appcmd.Command{
+		Use:   name + " <source>",
+		Short: "Create a repository webhook.",
+		Args:  cobra.ExactArgs(0),
+		Run: builder.NewRunFunc(
+			func(ctx context.Context, container appflag.Container) error {
+				return run(ctx, container, flags)
+			},
+			bufcli.NewErrorInterceptor(),
+		),
+		BindFlags: flags.Bind,
+	}
+}
+
+type flags struct {
+	WebhookEvent   string
+	OwnerName      string
+	RepositoryName string
+	CallbackURL    string
+}
+
+func newFlags() *flags {
+	return &flags{}
+}
+
+func (f *flags) Bind(flagSet *pflag.FlagSet) {
+	flagSet.StringVar(
+		&f.WebhookEvent,
+		webhookEventFlagName,
+		"",
+		"The event type to create a webhook for. The proto enum string value is used for this input (e.g. 'WEBHOOK_EVENT_REPOSITORY_PUSH').",
```

Comment:
Hm. I can fix it up a bit since the usage docs are kind of weirdly described:
>  buf beta registry webhook create <source> [flags]

Here's how it looks:
```
➜  buf git:(main) ✗ buflocal beta registry webhook create --help
Create a repository webhook.

Usage:
  buf beta registry webhook create <source> [flags]

Flags:
      --callback_url string   The url for the webhook to callback to on a given event.
      --event string          The event type to create a webhook for. The proto enum string value is used for this input (e.g. 'WEBHOOK_EVENT_REPOSITORY_PUSH').
  -h, --help                  help for create
      --owner string          The owner name of the repository to create a webhook for.
      --repository string     The repository name to create a webhook for.

Global Flags:
      --debug               Turn on debug logging.
      --log-format string   The log format [text,color,json]. (default "color")
      --timeout duration    The duration until timing out. (default 2m0s)
  -v, --verbose             Turn on verbose mode.```

Reply:
Great, we could add that URL to the `Create a repository webhook.` message?

Reply:
also lets explain what is `<source>` in the long description, or maybe rename it?